src/ms.tex

```diff
@@ -287,9 +287,60 @@ \subsection{Funding} \label{sec:project-funding}
 
 \secauthor{Aarya Patil}
 
-Summarize funding sources (Moore, NASA) and amounts and what this has been used
-for.
+% Summarize funding sources (Moore, NASA) and amounts and what this has been
+% used for.
+
+With the goal of establishing a sustainable financial model for the Astropy
+project, the Coordination Committee under the new governance charter (see \S
+\ref{sec:project-governance}) established a standing Finance committee.
+This committee overlooks the planning and disbursement of finances on behalf
+of the project. The Astropy project accepts funds from institutions as well as
+individuals through NUMFOCUS\footnote{NumFOCUS is a 501(c)(3) nonprofit that
+supports and promotes world-class, innovative, open source scientific computing.},
+the fiscal sponsor of the project. Previously, no direct financial support
+was available for the project development, and NUMFOCUS covered the incurred
+operational costs. However, transitioning to long-term financial stability meant
+having an influx of funds in the form of institutional support \citep{2016arXiv161003159M},
+and hence the Astropy project applied for and successfully acquired several grants.
```

Comment:
Is “several” == “four”? I feel like if we know there are four grants we can enumerate below, we should say that.